tests/etl/test_loaders.py

```
@@ -0,0 +1,59 @@
+import pandas as pd
+from types import SimpleNamespace
+from src.etl.loaders import PostgreSQLLoader, TableMappingLoader
+
+
+class MockDBManager:
+    def __init__(self):
+        self.tables = {}
+        self.conn = SimpleNamespace(closed=False)
+        self.cursor = SimpleNamespace()
+
+    def load_df_to_table(
```

Ruff (0.11.9)

Missing return type annotation for public function `load_df_to_table`

(ANN201)